pkgs/tools/misc/ufetch-nixos/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "ufetch-nixos";
+  src = ./ufetch-nixos;
+
+  phases = "installPhase";
+
+  installPhase = ''
+    mkdir -p $out
+    cp ${src} $out
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tiny system info for NixOS";
+    homepage = "https://gitlab.com/jschx/-/blob/master/ufetch-nixos";
```

Comment:
Hey there, thanks!
There were a few reasons: `ufetch` at the time I submitted the PR was hardcoded to be the crux script, which I found confusing when I ran it on my machine. 
The source has a `#!/bin/sh` shebang in each script, which to my understanding breaks in NixOS so I modified it.
Lastly, I just didn’t know how to do it since it’s my first PR to a package manager, let alone NixOS. 

I can update that URL, no problem.

Reply:
You can use `patchShebangs` function to patch shebangs to a bash in Nix store. But by default it is done automatically for all executable files in package:

https://github.com/NixOS/nixpkgs/blob/1e835d98c5d63a24a348e3510c8670ffca211392/pkgs/build-support/setup-hooks/patch-shebangs.sh#L1-L10